packages/xod-tabtest/src/Tabtest.re

```diff
@@ -111,7 +111,7 @@ module Bench = {
       probeMap: Map.String.empty,
     };
     Patch.listPins(patchUnderTest)
-    |. Pin.normalizeLabels
+    |. Pin.normalizeLabelsForCpp
```

Comment:
Are we forcing a xoder to enter escaped pin labels in the header row? I don’t think it’s a good idea